utils/mine-db-poll-intervals.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+function mine_data_for_poll_intervals {
+    res=$(echo "$1" \
+        | sed '/\(writes\|involuntary\)/s/$/\n\n/' \
+        | egrep -A 2 -B 1 'Unreasonably' \
+        | grep -v '\-\-' | cut -c 45- | paste -d " " - - - - \
+        | sed 's/.*took \([0-9]*\)ms .*long \([0-9]*\)ms .* \([0-9]*\)ms system.* \([0-9]*\) writes.*/\2 \3 \4 compaction \1/' \
+        | sed 's/.*long \([0-9]*\)ms .* \([0-9]*\)ms system.*/\1 \2/' \
+        | sort -h | grep -v compaction | less | cut -f 1 -d ' ' \
```

Comment:
Nit: `less` not needed.

Nit2: To some extent this is similar to `mine_data_for_poll_intervals()` from `mine-poll-intervals.sh`; I'd prefer not duplicating this complex data manipulation but if there's no easy way let's keep it as is for now.

Reply:
I'll remove `less`, thanks!  Construction of this pipeline was kind of iterative, so it seems like a leftover.  This function especially is a bit magical. :)
For the similarity, I agree that these functions duplicates each other, but I'm not sure what is a good way to break them into reusable modules.  The most I can do is to split the `datamash` part + check for zero out of them, I guess.  But, maybe, it's better to just keep them as is.  If we'll need more of these functions in the future,we can refactor that to a library of shell scripts later.

Reply:
Ok, let's keep them as is for now.